README.md

```diff
@@ -1,10 +1,20 @@
 # Overview
 
-This repository exists to share Github templates with other repositories within
-the the [`ChromaticHQ`](https://github.com/ChromaticHQ) organization.
+This repository exists to share Github templates and configuration with other
+repositories within the the [`ChromaticHQ`](https://github.com/ChromaticHQ)
+organization.
 
 **This repository is public**. It needs to be public for other repositories to
 be able to leverage the templates that it provides.
 
 The approach was taken from
 [this help page](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file).
+
+## Stalebot Configuration
```

Comment:
```suggestion
## Stalebot Configuration

```
Manual YML linting. 😂

Reply:
I see we have linting here that didn't catch this, but I thought other repos were flagging this. 🤷

Reply:
We have yaml linting on this repo, but not automated markdown linting.

Reply:
🤔 

There must be different rules in play then what I was linting with locally for markdown.